metron-platform/metron-pcap-backend/src/main/java/org/apache/metron/spout/pcap/deserializer/KeyValueDeserializer.java

```diff
@@ -28,6 +26,17 @@
 public abstract class KeyValueDeserializer implements Serializable {
   protected TimestampConverter converter;
 
+  public static class Result {
```

Comment:
Well, to be fair, it's `KeyValueDeserializer.Result` since it's a static inner class, not just `Result`.  I'm inclined to think the naming is fine given the parent class, but I'm open to being convinced otherwise.